### DISCUSSION TITLE
Macbook M1 Pro here, can't use

### DISCUSSION THREAD

**Discussion by ivyusa** [MRColorR/money4band#87] [Q&A] [answered]
hello, i'm on macbook pro m1 it's not work on automatic install
![Screenshot_2024-06-30_at_9 55 28_AM](https://github.com/MRColorR/money4band/assets/54756736/4722131d-e559-41f8-a5fc-aef9e3d28b94)
as this my shell is default, why it happens?
![Screenshot_2024-06-30_at_9 33 08_PM](https://github.com/MRColorR/money4band/assets/54756736/7945341d-acc6-4cc1-af84-992dcdf80615)
We had conversation on discord

**Comment by MRColorR** [comment]
thank you @ivyusa for reporting this also here on github :) 
it will help us keep it tracked

**Comment by MRColorR** [comment, marked as answer]
Money4Band v4 fixed this issue
